src/zope/testing/setupstack.txt

```diff
@@ -224,8 +224,7 @@ In addition to a tearDown method, the class provides methods:
 
 Here's an example:
 
-    >>> os.listdir('.') != []
-    True
+    >>> curdir = os.getcwd()
```

Comment:
FWIW we have a `here = os.getcwd()` earlier in this file, we could reuse it.

Reply:
Good point.  Done.